components/picker/index.css

```diff
@@ -24,7 +24,7 @@ governing permissions and limitations under the License.
 .spectrum-Picker {
   /* font */
   --spectrum-picker-font-size: var(--spectrum-font-size-100);
-  --spectrum-picker-placeholder-font-style-default: italic;
+  --spectrum-picker-placeholder-font-style-default: normal;
```

Comment:
If this is the _default_ and is also the CJK value, should we remove this as a modifier?

Reply:
Yeah, we can simplify this a lot now.